pyshelf/schema_validator.py

```diff
@@ -0,0 +1,33 @@
+from jsonschema import validate
+import json
+
+
+class SchemaValidator(object):
+    def __init__(self, logger):
+        self.logger = logger
+
+    def validate(self, data, schema_path):
+        """
+            Validates data against schema. This supresses all exceptions and merely returns True
+            on success and False on failure. It also logs the details of the exception.
+
+            Args:
+                data(type outlined by schema_name)
+                schema_path(string)
+
+            Returns:
+                bool: if data does not match schema
+        """
+        path = "schemas/{0}".format(schema_path)
```

Comment:
Use os.path.join instead for any path you need to create.  Makes it more flexible.